api/core/v2/rbac_test.go

```diff
@@ -197,15 +197,16 @@ func Test_split(t *testing.T) {
 
 func TestValidateSubjects(t *testing.T) {
 	tests := []struct {
-		Name     string
-		ExpErr   bool
-		Subjects []Subject
+		Name         string
+		ExpErr       bool
+		Subjects     []Subject
+		WantSubjects []Subject
 	}{
 		{
 			Name: "valid",
 			Subjects: []Subject{
 				{
-					Type: "user",
+					Type: "User",
```

Comment:
Should `"User"` be replaced with `corev2.UserType` to be consistent?

Reply:
Absolutely!